epi_scanner/app.py

```diff
@@ -133,12 +146,12 @@ async def update_weeks(q: Q):
         await q.page.save()
         q.page["plot_alt"] = ui.vega_card(
             box="week_map",
-            title="Number of weeks of Rt > 1 since 2010",
+            title="",
```

Comment:
I think this title must remain, because this information is not available elsewhere.